sphere-convenience/src/main/java/io/sphere/sdk/client/metrics/SimpleMetricsSphereClient.java

```diff
@@ -0,0 +1,50 @@
+package io.sphere.sdk.client.metrics;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientDecorator;
+import io.sphere.sdk.client.SphereRequest;
+
+import java.util.Observable;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * A decorator for {@link SphereClient}s which collects the time of serialization from and to JSON
+ * as well as the time waiting for the response of the commercetools platform.
+ *
+ * {@include.example io.sphere.sdk.client.metrics.SimpleMetricsSphereClientDemo}
+ */
+public final class SimpleMetricsSphereClient extends SphereClientDecorator implements SphereClient {
```

Comment:
Isn't it redundant to indicate that it's implementing `SphereClient` when `SphereClientDecorator` already does?